static/js/main.js

```diff
@@ -169,12 +170,46 @@ const add_awg_select_options = () =>{
     }
 };
 
+const graphRange = x_value => {
+  let ranges = [
+    {'name' : 'Voltage', 'min': '0', 'max': '50', 'dMin': '1', 'dMax': '30', 'step': '1',},
+    {'name' : 'Length', 'min': '0', 'max': '50', 'dMin': '10', 'dMax': '30', 'step':'1',},
+    {'name' : 'r0', 'min': '0', 'max': '8.0', 'dMin': '2.0', 'dMax': '5.0', 'step': '0.1',},
+    {'name' : 'ra', 'min': '0', 'max': '8.0', 'dMin': '3.0', 'dMax': '5.0', 'step': '0.1',},
+    {'name' : 'x', 'min': '0', 'max': '10', 'dMin': '0', 'dMax': '10', 'step': '1',},
+    {'name' : 'Force', 'min': '0', 'max': '50', 'dMin': '5', 'dMax': '20', 'step': '1',},
+    {'name' : 'AWG', 'min': '0', 'max': '40', 'dMin': '26', 'dMax': '40', 'step': '1',},
```

Comment:
I think we might want to allow up to 100 for (length, r0, ra, and x) unit types. This way it will work for any of the different unit types. Some would clearly be impossible like a 100ft long solenoid, but we probably don't want to waste time trying to manage and error check all of that.

Reply:
We also might want to think about allowing the user to pick a step value. Someone might want some finer grained charts for any of the values that are set to 1 right now.

Reply:
+1 on that